packages/pwa-kit-react-sdk/src/ssr/universal/components/refresh/index.jsx

```diff
@@ -39,6 +39,19 @@ const Refresh = () => {
                 console.warn('Could not find `referrer` search param - redirecting to home page.')
                 referrer = '/'
             }
+
+            // if experimentalUnsafeReloadServerSide is true, reload the page on server side
+            const experimentalUnsafeReloadServerSide = new URLSearchParams(location.search).get(
+                'experimentalUnsafeReloadServerSide'
+            )
+            if (experimentalUnsafeReloadServerSide === 'true') {
+                console.warn(
+                    '"experimentalUnsafeReloadServerSide" was set to true - reloading page on server side'
+                )
+                window.location.assign(referrer)
+                return
+            }
+
```

Comment:
Hey @sandragolden is there a chance that this code will stop stop SCAPI context from taking effect on the page? Or will there be a cserver refresh and a client refresh?

Reply:
Before this code runs, we should have already gotten a response back from the create-shopper-context endpoint, because Runtime Admin `await` the request to that endpoint and then send the `contextChange` event to the storefront.

Reply:
Thanks for the response. My concern is actually that if `experimentalUnsafeReloadServerSide` is true, then we are essentially bypassing the solution we added to ensure that we reload the latest data. For example, if `experimentalUnsafeReloadServerSide = true` and this runs, we would simply make another call to the server which always runs without context and the data isn't effected by the context.

Unfortunately there isn't a deployed version of this code for me to test and the PR test procedure is only on local host which won't actually make successful calls to the scapi context endpoint.

Reply:
I'm not sure I follow. `experimentalUnsafeReloadServerSide` = true simply refreshes the page. I could add the refresh on the runtime admin side, if that's what you mean. The shopper context isn't lost because it's already been applied to the shopper JWT and the refresh doesn't lose it.

this is deployed outside of localhost: https://runtime-admin-preview-sandra.mobify-storefront.com/kyle-preview-demo/p/test-sandra/preview?effectiveDateTime=1698816527069&path=/?vse=z4x2qptwzsz01brbyg1o5sp3v-gvzrfgnzc-1698816527069.staging.bigcontent.io%26vse-timestamp%3D1698816527069